### ISSUE TITLE
Guidance in Headings for top-level should be best practice 

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-web#2444]
## Describe the bug
 This sentence, "Use exactly one top-level heading. (WCAG 1.3.1)" is still on the "Guidance for Headings" page, but that is not a WCAG violation.

**To Reproduce**
Go to Assessment > Headings> Guidance >Do's

## Expected behavior

the "WCAG 1.3.1" in parentheses should be replaced with "best practice"

## Context (please complete the following information)

Accessibility Insights for Web
Version 2.16.0 | Powered by axe-core 3.5.1

## Additional context

reported by a user

**Comment by Shobhit1** [comment]
Fixed.

**Comment by ferBonnin** [comment]
Validated in Canary 2020.4.16.19

Thanks for the fix @Shobhit1!
